vstruct/defs/ihex.py

```diff
@@ -77,10 +77,7 @@ def vsEmit(self, fast=False):
             ffvals = [ ff.vsGetValue() for ff in self._vs_fastfields ]
             return struct.pack(self._vs_fastfmt, *ffvals)
 
-        ret = b''
-        for fname, fobj in self.vsGetFields():
-            ret += fobj.vsEmit() + b'\r\n'
-        return ret.decode('utf-8')
+        return b''.join(fobj.vsEmit() + b'\r\n' for _, fobj in self.vsGetFields())
```

Comment:
`b'\r\n'.join(fobs.vsEmit() for _, fobj in ...)`

perhaps? tiny bit faster, tiny bit shorter.

Reply:
make sense to me, I'll update it